Implementations pub - Added Document and image (Kit Product Creation) in bundles

prachi872 opened the pull request.

prachi872 committed:
kit product creation image

prachi872 committed:
Delete gorjana/.gitbook/assets/IMG_2942.heic

prachi872 committed:
Added file for kit product creation

prachi872 committed:
Update bundles.md

prachi872 committed:
Update bundles.md,added image for kit product creation

toaditi commented on gorjana/flows/products/bundles.md:
```diff
@@ -0,0 +1,58 @@
+
+### **Why This Transformation is Needed**
+
+NetSuite uses a unique Product ID, the "Internal ID," while the OMS relies on the OFBiz Product ID, which NetSuite doesn't recognize. To bridge this, the OMS stores the NetSuite Product ID in the *GoodIdentification* entity. NiFi maps the NetSuite ID to the OFBiz ID, ensuring seamless data synchronization.
```
Netsuite and Hotwax use Internal ID and Product ID respectively as product identifiers. HotWax stores the Internal ID in the GoodIdentification entity and then maps it to the Product ID

Update this to this sentence.

prachi872 replied:
Updated.

toaditi commented on gorjana/flows/products/bundles.md:
```diff
@@ -0,0 +1,58 @@
+### <a name="_5st0nlzeiktv"></a>**Creating Bundles at Gorjana**
+Previously, Gorjana used the Bundle App to manage bundle products, but with HotWax Commerce as their OMS, they now handle bundles seamlessly without the need for the app.
+#### <a name="_528yrqr0ttw5"></a>**1. Sourcing the Main Bundle from Shopify**
+Gorjana sources its main bundles, also known as parent products, directly from Shopify. In Shopify, bundles and finished goods are identical, so the OMS uses the same product import job, **IMPORT\_PRODUCTS,** to handle both seamlessly. Bundles are tagged as “Kit” in Shopify, with a unique SKU like any other product. Kit components, such as the earrings and necklace in a “Chloe Set” bundle, are ta ...
```
remove 'like any other product'

prachi872 committed:
Update bundles.md

Added - Netsuite and Hotwax use Internal ID and Product ID respectively as product identifiers. HotWax stores the Internal ID in the GoodIdentification entity and then maps it to the Product ID
Removed - "Any other product"

prachi872 committed:
Add files via upload

prachi872 committed:
Update and rename Kit inventory Computation F.md to Kit inventory Computation .md
